src/encoded/auth0.py

```diff
@@ -96,6 +93,40 @@ def forget(self, request):
         return []
 
 
+def signup(request):
+    """
+    Create new user.
+
+    :param request: Pyramid request object
+    """
+    domain = 'encode.auth0.com'
+    url = "https://{domain}/userinfo?access_token={access_token}".format(domain=domain, access_token=request.json['accessToken'])
```

Comment:
What if `accessToken` isn't in JSON?

Reply:
Let me think about that a little. My idea is to let the code blow up as the user has an invalid request object. This is the way it currently works.